### DISCUSSION TITLE
Gemini support

### DISCUSSION THREAD

**Discussion by DonPiotr** [gsantner/markor#1910] [Ideas] [answered]
Hi. I think that adding to Markor possibility to edit .gmi files would be nice feature.

**Comment by gsantner** [comment]
I don't know what Gemini is, but if it's a plaintext format feel free to implement it and make a merge request.

**Reply by DonPiotr** [reply, marked as answer]
Sorry, I forgot that Gemini protocol is extremely nerdy thing. You can look hear: https://gemini.circumlunar.space/
*.gmi files are much simpler (with less features) then markdown.
Implement by myself won't be easy. I have mach more experience with system administration then coding and I have no experience at all with Java or with coding for Android. 😞

**Reply by gsantner** [reply]
@DonPiotr   
I don't use Gemini and I don't think about starting to use it. So I won't implement it. If you want it, please feel free to start contributing - and if you are new to Android/Java that shouldn't be a big problem, everything can be learned, as well changes can be done one step a time.
